extensions/amp-access/0.1/test/test-access-expr.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {evaluateAccessExpr} from '../access-expr';
+import * as sinon from 'sinon';
+
+
+describe('evaluateAccessExpr', () => {
+
+  it('should evaluate simple boolean expressions', () => {
+    expect(evaluateAccessExpr('access', {})).to.be.false;
+
+    expect(evaluateAccessExpr('access', {access: true})).to.be.true;
+    expect(evaluateAccessExpr('access', {access: false})).to.be.false;
+
+    expect(evaluateAccessExpr('access', {access: 1})).to.be.true;
+    expect(evaluateAccessExpr('access', {access: 0})).to.be.false;
+
+    expect(evaluateAccessExpr('access', {access: '1'})).to.be.true;
```

Comment:
That's not fully decided yet. However, on the surface, I don't see a bit reason not to do so. E.g. an expression could be built on top of "viewsLeft" number that is either `> 0` or `0`. Eventually we will allow expressions such as `viewsLeft > 0`, but we are not there quite yet.